wiki/Tournaments/HC/OMEGA/en.md

```diff
@@ -0,0 +1,235 @@
+- Group stage: No best of. Play with the intent to get the highest score.
+- 1v1 bracket:
+  - Round of 16 and Quarterfinals: Best of 7.
+  - Semifinals: Best of 9.
+  - Grand Finals: Best of 11.
+
+### Scheduling
+
+1. The schedule for the tournament is listed under [tournament scheduling](#tournament-schedule), please use the time in-between to practice the mappool(s).
+2. Matches are allowed to overlap if referees are available.
+3. For 1v1 Bracket:
+   - If both players can come up with a better time than what is scheduled, please inform the organisers as soon as possible so they can make adjustments.
+
+### Mappools
+
+1. For more specifics on what maps were played see: [Mappools](#mappools)
+2. There will be a variety of *Harumachi Clover* maps to choose from in each stage of the tournament.
+   - Note: 1v1 bracket stage **does not** allow bans. See: [Number 4](#tournament-staging)
+3. Each stage will have the number of maps and mods as seen below:
+   - Qualifiers/Groups: 1 NM, 1 HD, 1 HR, 1 DT, 1 FM
+   - RO16/Quarterfinals: 2 NM, 1 HD, 1 HR, 1 DT, 1 FM, 1 TB
+   - Semifinals: 2 NM, 1 HD, 1 HR, 2 DT, 2 FM, 1 TB
+   - Grand Finals: 2 NM, 2 HD, 2 HR, 2 DT, 2 FM, 1 TB
+
+### Seeding specifics
+
+1. Seeding will take place during the Qualifiers and the Group Heat stages.
+2. During the Qualifiers Stage there will be multiple lobbies open for sign-ups. Please make sure to sign up for a time that best fits your schedule.
+3. The Qualifiers Stage is used to help get rid of those who don't show up and to get the participation count to the nice number of 64.
+4. There will be no warmups for the Qualifiers and Group Heats stages, remember this is a *Harumachi Clover* tournament, the maps are only 40 seconds at most.
+5. Seeding will be gathered by taking the highest score recorded and used in comparison to the scores of the other players.
+
+### Lobby specifics
+
+1. A private match will be created for the matches. The players will be invited to the asscociated match around 15 minutes before the match time. Make sure to have PMs enabled for everyone not just on your friend's list as the invite does **not** bypass this setting. If you are unable to make it to the match due to this issue, your position in the tournament is **not** guaranteed.
+2. During the 1v1 Bracket, each player will `!roll`, the winner of the roll **picks who will pick the first map**. Tournament staff would like to preface, that if you win the `!roll`, this does **not** mean you pick the first map, this means you pick whether you pick the first map or your opponent does. Choose wisely!
+3. For the Qualifiers and Group Heats stage, **the mappool will be played through twice**.
+4. Players will be allowed to ask for a brief break at the end of the first run-through of the mappool.
+5. There are **no** bans.
+6. There are **no** warmups. Be prepared.
+7. 1v1 Bracket only: In case of a tie, the tiebreaker will be played.
+   - Otherwise, see: [Number 6](#ruleset).
```

Comment:
```suggestion
7. 1v1 bracket only: in case of a tie, the tiebreaker will be played. See [rule 6 of the general rules](#general) for more details.
```